docs/runtimes/node-affinity.md

```diff
@@ -1,96 +1,214 @@
-# Node affinity for MLRun jobs
-Node affinity can be applied to MLRun to determine on which nodes 
-they can be placed. The rules are defined using custom labels on nodes and label selectors. 
-Node affinity allows towards Spot or On Demand groups of nodes.
+# Managing job resources
+
+Configuration of job resources is relevant for all supported cloud platforms.
```

Comment:
Need to start with some overview paragraph, e.g: 

MLRun orchestrate serverless functions over Kubernetes, the user can specify desired resource requirements (CPU, memory, GPUs) and preferences/priorities in the logical function object, this will be used during the function deployment.